Feat: show airdrop claim status and filter by claimed/unclaimed (#6565)
rotki/rotki: docs/changelog.rst

Changelog
=========

> * :feature:`-` Users will now be able to filter claimed and inclaimed airdrops.

Reviewer: ```suggestion
* :feature:`-` Users will now have the ability to filter between claimed and unclaimed airdrops.
```
